packages/facility-server/app/routes/apiv1/encounter.js

```diff
@@ -555,21 +555,7 @@ encounterRelations.get(
 
     // Convert to plain object to avoid circular references
     const invoice = invoiceRecord.get({ plain: true });
-
-    const invoiceItemsResponse = invoice.items.map(item => {
-      const itemInsurancePlansById = keyBy(item.product?.invoiceInsurancePlanItems, 'invoiceInsurancePlanId');
-
-      const insurancePlanItems =
-        invoice?.insurancePlans?.map(({ id, code, name, defaultCoverage }) => {
-          const invoiceItem = itemInsurancePlansById[id];
-          const coverageValue = invoiceItem?.coverageValue ?? defaultCoverage;
-          const label = name || code;
-          return { id, code, name, label, coverageValue };
-        }) || [];
-
-      return { ...item, insurancePlanItems };
-    });
-
+    const invoiceItemsResponse = invoice.items.map(getInsurancePlanItems(invoice.insurancePlans));
```

Comment:
split out the getInsurancePlanItems logic so that it could also be used in the patient invoices controller